compute_benchmark_age_prediction.py

```diff
@@ -260,7 +264,14 @@ def run_benchmark_cv(benchmark, dataset):
             f"'{benchmark}' on dataset '{dataset}'")
         return
 
-    metrics = [mean_absolute_error, r2_score]
+    ys_true, ys_pred = [], []
+
+    def mean_absolute_error_with_memory(y_true, y_pred):
+        ys_true.append(y_true)
+        ys_pred.append(y_pred)
+        return mean_absolute_error(y_true, y_pred)
```

Comment:
@dengemann this is the trick I used to avoid running cross_val_predict

Reply:
Smart! I love it!!!